### PR TITLE
SYNCT-171: Exposed methods for manual push/pull records to/from parent

### PR DESCRIPTION
**Pull request by oskarhinc**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/sync2/api/service/impl/SyncPullServiceImpl.java [hunk 1]
```diff
@@ -86,6 +91,11 @@ public AuditMessage pullDataFromParentAndSave(String category, Map<String, Strin
         return auditMessage;
     }
 
+    @Override
+    public void manuallyPullFromParent(String category) throws SyncException {
+        parentFeedReader.readFeedsForPull(category);
```
Comment (areklalo) [inline api/src/main/java/org/openmrs/module/sync2/api/service/impl/SyncPullServiceImpl.java]: @oskarhinc 
Do we only read the feeds from the parent?

File: api/src/main/java/org/openmrs/module/sync2/api/service/impl/SyncPushServiceImpl.java [hunk 2]
```diff
@@ -80,7 +86,12 @@ public AuditMessage readDataAndPushToParent(String category, Map<String, String>
         }
         return auditMessage;
     }
-    
+
+    @Override
+    public void manuallyPushToParent(String category) throws SyncException {
+        localFeedReader.readFeedsForPush(category);
```
Comment (areklalo) [inline api/src/main/java/org/openmrs/module/sync2/api/service/impl/SyncPushServiceImpl.java]: Same as above

File: api/src/main/java/org/openmrs/module/sync2/api/service/SyncPullService.java [hunk 3]
```diff
@@ -6,11 +6,11 @@
 
 public interface SyncPullService {
 
-    AuditMessage pullDataFromParentAndSave(String category, Map<String, String> resourceLinks,
+    AuditMessage pullAndSaveFeedsFromParent(String category, Map<String, String> resourceLinks,
```
Comment (areklalo) [inline api/src/main/java/org/openmrs/module/sync2/api/service/SyncPullService.java]: This method will pull and save not atom feeds but the single OpenmrsObject (using atom feed).

File: api/src/main/java/org/openmrs/module/sync2/api/service/SyncPushService.java [hunk 4]
```diff
@@ -6,10 +6,10 @@
 
 public interface SyncPushService {
 
-    AuditMessage readDataAndPushToParent(String category, Map<String, String> resourceLinks, String action);
+    AuditMessage readAndPushFeedsToParent(String category, Map<String, String> resourceLinks, String action);
```
Comment (areklalo) [inline api/src/main/java/org/openmrs/module/sync2/api/service/SyncPushService.java]: Same as above

File: api/src/main/java/org/openmrs/module/sync2/api/service/SyncPullService.java (hunk 3 above)
Comment (oskarhinc) [reply api/src/main/java/org/openmrs/module/sync2/api/service/SyncPullService.java]: true

**Pull request merged**